atomsci/ddm/utils/struct_utils.py

```diff
@@ -342,9 +340,9 @@ def _standardize_chemistry(df, standard='rdkit', smiles_col='rdkit_smiles', work
             try:
                 mol = Chem.MolFromSmiles(smi)
                 out.append(Chem.inchi.MolToInchi(mol))
-            except:
+            except Exception:
                 out.append('Invalid SMILES: %s' % (smi))
-    elif std == 'name':
+    elif standard.lower() == 'name':
```

Comment:
would have broken if input was not inchi or rdkit; needs test

Reply:
This function is only called by _merge_dataframes_by_smiles which is deprecated and isn't used anywhere.